solutions/swb-ui/infrastructure/.eslintignore

```diff
@@ -0,0 +1,7 @@
+.eslintrc.js
+node_modules
+lib
+temp
+cdk.out
+build
+src/redirectFunction.js
```

Comment:
reason why we're ignoring this file?

Reply:
This conversation is resolved, answered on slack !